main.go

```diff
@@ -883,6 +884,13 @@ func (task *TaskRun) run() error {
 	if err != nil {
 		return WorkerShutdown(err)
 	}
+	// using NewWriterLevel(...) instead of NewWriter(...) to be explicit about
+	// compression level, so easy to change later
+	gzipLogWriter, err := gzip.NewWriterLevel(logFileHandle, gzip.DefaultCompression)
```

Comment:
Yeah ok I'll switch back. Originally I actually had it on `BestCompression` (hard coded) which worked fine, except that `gunzip` on my mac wasn't able to decompress it, so I switched to the default level. Now I'm doing that, I agree with you, it is no doubt simpler to use `NewWriter` instead.